src/mediapipe_internal/mediapipegraphdefinition.cpp

```diff
@@ -438,22 +438,38 @@ std::pair<std::string, mediapipe_packet_type_enum> MediapipeGraphDefinition::get
     return {"", mediapipe_packet_type_enum::UNKNOWN};
 }
 
+struct PythonResourceInitializationCleaningGuard {
```

Comment:
Maybe we can shorten this: 
PythonResourceInitializationCleaningGuard -> PythonResourcesCleaningGuard

Reply:
done